odo/backends/csv.py

```diff
@@ -368,7 +369,7 @@ def CSV_to_chunks_of_dataframes(c, chunksize=2 ** 20, **kwargs):
     else:
         rest = []
 
-    data = [first] + rest
+    data = chain([first], rest)
```

Comment:
`toolz.chain` should be able to do this, no?  It's already imported, too.

Reply:
there is no toolz.chain. similar functions in toolz just import itertools.
toolz source:
```
def concat(seqs):
    """ Concatenate zero or more iterables, any of which may be infinite.

    An infinite sequence will prevent the rest of the arguments from
    being included.

    We use chain.from_iterable rather than ``chain(*seqs)`` so that seqs
    can be a generator.

    >>> list(concat([[], [1], [2, 3]]))
    [1, 2, 3]

    See also:
        itertools.chain.from_iterable  equivalent
    """
    return itertools.chain.from_iterable(seqs)



def concatv(*seqs):
    """ Variadic version of concat

    >>> list(concatv([], ["a"], ["b", "c"]))
    ['a', 'b', 'c']

    See also:
        itertools.chain
    """
    return concat(seqs)

```

Reply:
Sorry, typo; I meant `toolz.concat`.

Reply:
do you want to change it?